Cmr output

brettz9 opened the pull request:
Fixes #61. Builds on #62 and #63.

- Enhancement: Add cmrOutput reporter option for dynamic output

YOU54F commented:
Could we document the new option please? Once we’ve rebased, I can merge this and get a release out

brettz9 commented:
Sure, I've documented and rebased.

I also realized this should have been more robust by letting one specify the specific reporter (and multiple ones if desired) along with the specific reporter property, so I've adjusted accordingly.

brettz9 commented:
Sorry, had some fixes to avoid separator characters that had special meaning in the shell; now applied and tested in another repo.
